{
	"id": "openSUSE-SU-2019:2259-1",
	"modified": "2019-10-06T08:20:28Z",
	"published": "2019-10-06T08:20:28Z",
	"related": [
		"CVE-2019-6446"
	],
	"summary": "Security update for python-numpy",
	"details": "This update for python-numpy fixes the following issues:\n\nNon-security issues fixed:\n\n- Updated to upstream version 1.16.1. (bsc#1149203) (jsc#SLE-8532)\n\nThis update was imported from the SUSE:SLE-15:Update update project.\nThis update was imported from the openSUSE:Leap:15.1:Update update project.",
	"affected": [
		{
			"package": {
				"ecosystem": "SUSE:Package Hub 15 SP1",
				"name": "python-numpy",
				"purl": "purl:rpm/suse/python-numpy&distro=SUSE%20Package%20Hub%2015%20SP1"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1.16.1-bp151.2.3.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python2-numpy-gnu-hpc": "1.16.1-bp151.2.3.1",
						"python2-numpy-gnu-hpc-devel": "1.16.1-bp151.2.3.1",
						"python2-numpy_1_16_1-gnu-hpc": "1.16.1-bp151.2.3.1",
						"python2-numpy_1_16_1-gnu-hpc-devel": "1.16.1-bp151.2.3.1",
						"python3-numpy-gnu-hpc": "1.16.1-bp151.2.3.1",
						"python3-numpy-gnu-hpc-devel": "1.16.1-bp151.2.3.1",
						"python3-numpy_1_16_1-gnu-hpc": "1.16.1-bp151.2.3.1",
						"python3-numpy_1_16_1-gnu-hpc-devel": "1.16.1-bp151.2.3.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "SUSE:Package Hub 15 SP1",
				"name": "python-numpy_1_16_1-gnu-hpc",
				"purl": "purl:rpm/suse/python-numpy_1_16_1-gnu-hpc&distro=SUSE%20Package%20Hub%2015%20SP1"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1.16.1-bp151.2.3.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python2-numpy-gnu-hpc": "1.16.1-bp151.2.3.1",
						"python2-numpy-gnu-hpc-devel": "1.16.1-bp151.2.3.1",
						"python2-numpy_1_16_1-gnu-hpc": "1.16.1-bp151.2.3.1",
						"python2-numpy_1_16_1-gnu-hpc-devel": "1.16.1-bp151.2.3.1",
						"python3-numpy-gnu-hpc": "1.16.1-bp151.2.3.1",
						"python3-numpy-gnu-hpc-devel": "1.16.1-bp151.2.3.1",
						"python3-numpy_1_16_1-gnu-hpc": "1.16.1-bp151.2.3.1",
						"python3-numpy_1_16_1-gnu-hpc-devel": "1.16.1-bp151.2.3.1"
					}
				]
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/2OYZN2KXJT6YZ75UZQ32FP2DTHA7CNTW/#2OYZN2KXJT6YZ75UZQ32FP2DTHA7CNTW"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1149203"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2019-6446"
		}
	]
}
